protos/local/temporal/sdk/core/workflow_activation/workflow_activation.proto

```diff
@@ -242,6 +245,29 @@ message ResolveRequestCancelExternalWorkflow {
     temporal.api.failure.v1.Failure failure = 2;
 }
 
+// Lang is requested to invoke an update handler on the workflow. Lang should invoke the update
+// validator first (if requested). If it accepts the update, immediately invoke the update handler.
+// Lang must reply to the activation containing this job with an `UpdateResponse`.
+message DoUpdate {
+    // A workflow-unique identifier for this update
+    string id = 1;
```

Comment:
I am assuming there is no need to surface this to the user? I wonder if we need to expose to lang.

Reply:
Yes, I think they'll expect/want to have it.